packages/manager/modules/anthos/src/anthos.component.js

```diff
@@ -0,0 +1,13 @@
+import template from './anthos.html';
+import controller from './anthos.controller';
+
+export default {
+  template,
+  controller,
+  bindings: {
+    user: '<',
+    guides: '<',
```

Comment:
We have the `guides` binding. But I see that you get the guides only in the controller. Either this can be removed or you can retrieve the guides in the routes

Reply:
If i need it in another place, i will add it.
Thanks.